test/test_fmm.py

```diff
@@ -691,27 +682,24 @@ def test_sumpy_target_point_multiplier(ctx_factory, deriv_axes):
     from boxtree.fmm import drive_fmm
 
     pot0, pot1, pot2 = drive_fmm(wrangler, (weights,))
-    pot0, pot1, pot2 = pot0.get(), pot1.get(), pot2.get()
+    pot0, pot1, pot2 = actx.to_numpy(pot0), actx.to_numpy(pot1), actx.to_numpy(pot2)
     if deriv_axes == (0,):
-        ref_pot = pot1 * sources[0].get() + pot2
+        ref_pot = pot1 * actx.to_numpy(sources[0]) + pot2
     else:
-        ref_pot = pot1 * sources[0].get()
+        ref_pot = pot1 * actx.to_numpy(sources[0])
 
     rel_err = la.norm(pot0 - ref_pot) / la.norm(ref_pot)
     logger.info("order %d -> relative l2 error: %g", order, rel_err)
 
     assert np.isclose(rel_err, 0, atol=1e-5)
 
+# }}}
 
-# You can test individual routines by typing
-# $ python test_fmm.py 'test_sumpy_fmm(cl.create_some_context, LaplaceKernel(2),
-#       VolumeTaylorLocalExpansion, VolumeTaylorMultipoleExpansion, False, False)'
```

Comment:
Can you keep these lines and change `cl.create_some_context` to `_acf`? I use this quite frequently.

Reply:
Yeah, definitely!

Reply:
This should be back in the latest version, plus updated the comment in the other test files as well.

Thanks for taking a look and let me know if you spot anything else that's fishy!